component/vshn_alerting.jsonnet

```diff
@@ -4,6 +4,69 @@ local inv = kap.inventory();
 local params = inv.parameters.appcat;
 local vars = import 'config/vars.jsonnet';
 
+local getRedisHARules(serviceName) = [
+  {
+    alert: 'VSHNRedisNotMaster',
+    annotations: {
+      description: 'Service {{ $labels.service }} is not connected to a master for 5m.',
+      summary: 'Redis not master: {{ $labels.name }} ({{ $labels.namespace }})',
+    },
+    expr: 'appcat_probes_redis_ha_master_up{ha="true"} == 0',
+    'for': '5m',
+    labels: {
+      OnCall: '{{ if eq $labels.sla "guaranteed" }}true{{ else }}false{{ end }}',
+      runbook: 'https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatRedis.html',
+      service: serviceName,
+      severity: 'critical',
+      syn: 'true',
+      syn_team: 'schedar',
+      syn_component: 'appcat',
+    },
+  },
+  {
+    alert: 'VSHNRedisQuorumNotOk',
+    annotations: {
+      description: 'Quorum failing for 5m.',
+      summary: 'Redis quorum not OK: {{ $labels.name }} ({{ $labels.namespace }})',
```

Comment:
Does `$labels.namespace` point to the instance namespace or the namespace where the alert is deployed?

Reply:
`$labels.namespace`contains the claim namespace

<img width="1288" height="427" alt="image" src="https://github.com/user-attachments/assets/a6de2e7b-63c5-428a-8bb6-4ebb184d9b65" />

Reply:
Can you switch that to the instance namespace? The claim namespace doesn't help very much. You'll have to dig out the instance namespace manually then...